tests/slr_integration/__snapshots__/Frontend_Test__should_replace_ticket_block_when_seating_is_enabled__ticket with future and past__0.snapshot.html

```diff
@@ -0,0 +1,47 @@
+<div  class="tribe-common event-tickets tribe-tickets__tickets-wrapper" >
+	<form
+		id="tribe-tickets__tickets-form"
+		action="http://wordpress.test/?tec-tc-cart=redirect"
+		class="tribe-tickets__tickets-form tribe-tickets__form"
+		method="post"
+		enctype='multipart/form-data'
+		data-provider="TEC\Tickets\Commerce\Module"
+		autocomplete="off"
+		data-provider-id="tickets-commerce"
+		data-post-id="{{post_id}}"
+		novalidate
+	>
+
+		<input type="hidden" name="tribe_tickets_saving_attendees" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar_data" value="" id="tribe_tickets_block_ar_data"/>
+
+		<input name="provider" value="TEC\Tickets\Commerce\Module" class="tribe-tickets-provider" type="hidden">
+
+		<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__tickets-title">
+	Tickets</h2>
+
+		<div id="tribe-tickets__notice__tickets-in-cart"  class="tribe-tickets__notice tribe-tickets__notice--barred tribe-tickets__notice--barred-left" >
+	
+	<div  class="tribe-common-b2 tribe-tickets-notice__content tribe-common-b3" >
+		The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.	</div>
+</div>
+
+		
+		
+		<div  class="tribe-tickets__tickets-item tribe-tickets__tickets-item--inactive" >
+	<div
+		class="tribe-tickets__tickets-item-content tribe-tickets__tickets-item-content--inactive"
+	>
+		There are no Tickets available at this time.	</div>
```

Comment:
I would expect `Tickets will be available on January 1, 2044` instead of that. Not blocking though since we are not introducing that.

Reply:
Yes, this is pre-existing for the default ET and this only shows up when there are past and future both tickets available but nothing for the current time.